src/redux/categories/categoriesSlice.js

```diff
@@ -0,0 +1,14 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const initialState = 'Under construction';
+
+const categoriesSlice = createSlice({
+  name: 'categories',
+  initialState,
+  reducers: {
+    checkStatus: (state) => state,
+  },
```

Comment:
- [ ] Great effort in adding the `checkStatus` reducer! 👏 However, there's a small adjustment needed to ensure it fulfills its intended purpose effectively.

  - The `checkStatus` reducer should primarily focus on examining the payload of the action it receives. If the payload matches the string `"Under construction"`, then it should update the "categories" state accordingly. In all other cases, it should leave the state unchanged.

To achieve this, consider using an `if` and `else` statement in your reducer logic. Here's how you can implement it:
```
checkStatus: (state, action) => {
      state.categories =
        action.payload === "Under construction"
          ? "Under construction"
          : state.categories
    },
```

This way, your reducer will accurately update the "categories" state to "Under construction" only when the payload matches that string, and it will preserve the current state for all other cases. 😊👍

Happy coding! 😊🚀🍀